### PR TITLE
Feature/Add toast visibility parameter

### PR DESCRIPTION
**Pull request by juanpiquerez**
Added in the paramOptions a new option called `hideToast` which allows hiding a specific type of toast notification. To hide multiple toast types, separate them using `,`. Possible values: `success`, `error`, `warning`, `info`.

### CODE REVIEW

File: src/components/VideoPlayerSideVideoSources.vue [hunk 1]
```diff
@@ -148,13 +148,9 @@ export default {
         try {
           await selectSource({ kind: 'audio', source: audioSource })
         } catch (error) {
-          this.toast.error(
-            'There was an error selecting the desired source, try again',
-            { timeout: 5000 }
-          )
+          this.toast.showToast('success', 'There was an error selecting the desired source, try again', { timeout: 5000 })
```
Comment (R-Delfino95) [inline src/components/VideoPlayerSideVideoSources.vue:151]: Should be error type, right?

**Review by R-Delfino95 — CHANGES_REQUESTED** [review]

**Review by R-Delfino95 — APPROVED** [review]

**Review by tatiana-perera — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -70,6 +70,7 @@ You can now use the plugin in any Vue file of your project as a component.
 | `environment`  | `Object`         | Optional  | `.env file content` | Plugin environment. See [Environment options](#environment-options) on how to configure.             |
 | `mainLabel`    | `String`         | Optional  |                     | Allows to change the label of the main video.                                                        |
 | `startingQuality` | `String` | Optional | Allows to start the stream at a specific resolution when available. Possible values: 'High', 'Medium', 'Low', <Number> specifying the desired frame height (i.e. 360). |
+| `hideToast`   | `String`          | Optional |  `null`              | Allows you to hide a specific type of toast, and you can hide more than one by separating them by ",". Possible values: 'success', 'error', 'warning', 'info'. |
```
Comment (tatiana-perera) [inline README.md:73]:
```suggestion
| `hideToast`   | `String`          | Optional |  `null`              | Allows to hide a specific type of toast, with the capability to hide multiple types simultaneously by `,`. Accepted values: 'success', 'error', 'warning', 'info'. |
```

File: README.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ You can now use the plugin in any Vue file of your project as a component.
 | `environment`  | `Object`         | Optional  | `.env file content` | Plugin environment. See [Environment options](#environment-options) on how to configure.             |
 | `mainLabel`    | `String`         | Optional  |                     | Allows to change the label of the main video.                                                        |
 | `startingQuality` | `String` | Optional | Allows to start the stream at a specific resolution when available. Possible values: 'High', 'Medium', 'Low', <Number> specifying the desired frame height (i.e. 360). |
```
Comment (tatiana-perera) [inline README.md:72]:
```suggestion
| `startingQuality` | `String` | Optional | `null`| Allows to start the stream at a specific resolution when available. Possible values: 'High', 'Medium', 'Low', <Number> specifying the desired frame height (i.e. 360). |
```

File: README.md [hunk 4]
```diff
@@ -69,7 +69,8 @@ You can now use the plugin in any Vue file of your project as a component.
...
 | `environment`  | `Object`         | Optional  | `.env file content` | Plugin environment. See [Environment options](#environment-options) on how to configure.             |
 | `mainLabel`    | `String`         | Optional  |                     | Allows to change the label of the main video.                                                        |
-| `startingQuality` | `String` | Optional | Allows to start the stream at a specific resolution when available. Possible values: 'High', 'Medium', 'Low', <Number> specifying the desired frame height (i.e. 360). |
+| `startingQuality` | `String` | Optional | `null`| Allows to start the stream at a specific resolution when available. Possible values: 'High', 'Medium', 'Low', <Number> specifying the desired frame height (i.e. 360). |
+| `hideToast`   | `String`          | Optional |  `null`              | Allows to hide a specific type of toast, with the capability to hide multiple types simultaneously by `,`. Accepted values: 'success', 'error', 'warning', 'info'. |
```
Comment (tatiana-perera) [inline README.md:73]:
```suggestion
| `hideToast`   | `String`          | Optional |  `null`              | Allows to hide a specific type of toast notification. To hide multiple toast types, separate them using `,`. Possible values: `success`, `error`, `warning`, `info`. |
```

**Pull request merged**